reserver/reserver_obj.py

```diff
@@ -29,7 +29,19 @@ def __init__(self, api_token, test_pypi=False):
         self.password = api_token
         self.test_pypi = test_pypi
 
-        
+
+    def batch_upload_to_pypi(self, *names):
+        """
+        Upload batch of package names to PyPI.
+
+        :param names: packages' names
+        :type names: vararg
+        :return: None
+        """
+        for name in names:
+            self.upload_to_pypi(name)
+
+
```

Comment:
If you accept my comments on returning `True` for `upload_to_pypi`. This can also return a logical AND of all uploading trials.

Reply:
Well, in my opinion, it is way too geeky, I prefer to return the number of actually reserved names here.